applications/callflow/doc/cf_branch_property.md

```diff
@@ -0,0 +1,42 @@
+/*
+Section: Callflows
+Title: Branch on callee's document property value
+Language: en-US
+Version: 4.0
+*/
+
+The `branch_property` callflow enables you to branch on value of some field inside callee's document (device, user or account).
+
+### Mandatory fields
+
+**property** - name of the property we are looking for
+
+### Optional fields
+
+**action** - "branch" - the only action supported now
```

Comment:
@lazedo, the customer is stating there's some more detail in the ticket comments regarding branch_property: https://2600hz.atlassian.net/browse/KAZOO-4601 (look at extreme rotary's comments)